tests/RobotFramework/tests/tedge_agent/http_server/entity_store_api.robot

```diff
@@ -58,3 +119,9 @@ Entity auto-registration over MQTT
 Custom Setup
     ${DEVICE_SN}=    Setup
     Set Suite Variable    $DEVICE_SN
+
+Register entity
+    [Arguments]    ${topic_id}    ${type}    ${parent}
+
+    Execute Command
+    ...    curl -X POST http://localhost:8000/tedge/entity-store/v1/entities -H 'Content-Type: application/json' -d '{"@topic-id": "${topic_id}", "@type": "${type}", "@parent": "${parent}"}'
```

Comment:
It might be worth adding this as a keyword to ThinEdgeIO.py

E.g. `Register Entity`